src/nix/describe_derivation.rs

```diff
@@ -126,6 +126,7 @@ pub fn describe_derivation(args: &DescribeDerivationArgs) -> Result<DerivationDe
         .arg(format!("lib={}", args.lib.path().to_string_lossy()))
         .args(["--json", "--expr", expr])
         .arg("--impure")
+        .args(["--extra-experimental-features", "flakes nix-command"])
```

Comment:
By the way, I haven't looked at it, this [Rust API for the Nix CLI](https://crates.io/crates/runix) was presented at NixCon two years ago I believe by Flox. Don't know if it's really helpful here, but I'm mentioning it just in case.

Reply:
It isn't unfortunately! When I started this project, I investigated it, and there was actually a prototype that used it. But I found it not powerful enough to do what we needed to do with it unfortunately.